buffer/src/main/java/io/netty/buffer/AbstractByteBuf.java

```diff
@@ -33,6 +33,7 @@
 import java.nio.channels.ScatteringByteChannel;
 import java.nio.charset.Charset;
 
+import static io.netty.util.internal.MathUtil.isOutOfBounds;
```

Comment:
Well it is not really related only to arrays as well as we use it in our direct buffers too :) So maybe it's just fine as MathUtil is an internal class anyway and we can move it later.